src/types.ts

```diff
@@ -32,3 +33,27 @@ export type MintOrBurnParams = {
   isMint: boolean;
   marginDelta: BigNumberish;
 };
+
+export enum SupportedNetworksEnum {
```

Comment:
Based on my research and the values that are returned by Metamask when requesting the chain id, what we get is '0x1' instead of 1. I suggest we update the enum to match this. Also I think it would make sense if we change the name from SupportedNetworksEnum to SupportedChainsEnum to match the idea behind the enum.